modules/core/src/shaderlib/index.js

```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {registerShaderModules, ProgramManager} from '@luma.gl/core';
-import {fp32, gouraudlighting, phonglighting} from '@luma.gl/core';
+import {ProgramManager} from '@luma.gl/core';
+import {gouraudlighting as gouraudLighting, phonglighting as phongLighting} from '@luma.gl/core';
```

Comment:
@tsherif Can we rename these at the source?

Reply:
Yes, definitely.